packages/admin-panel/src/pages/resources/MapOverlaysPage.js

```diff
@@ -105,9 +117,9 @@ const IMPORT_CONFIG = {
 };
 
 const renderNewMapOverlayVizButton = () => (
-  <Link to="/viz-builder/map-overlay/new">
+  <StyledLink to="/viz-builder/map-overlay/new">
```

Comment:
Note that we gotta make this same change to the `New` button in dashboard items as well 👍